cmd/template/cluster/flag.go

```diff
@@ -110,12 +121,12 @@ func (f *flag) Validate() error {
 		// Validate Master AZs.
 		switch f.Provider {
 		case key.ProviderAWS:
-			if len(f.MasterAZ) != 0 && len(f.MasterAZ) != 1 && len(f.MasterAZ) != 3 {
-				return microerror.Maskf(invalidFlagError, "--%s must be set to either one or three availability zone names", flagMasterAZ)
+			if len(f.ControlPlaneAZ) != 0 && len(f.ControlPlaneAZ) != 1 && len(f.ControlPlaneAZ) != 3 {
+				return microerror.Maskf(invalidFlagError, "--%s must be set to either one or three availability zone names", flagControlPlaneAZ)
```

Comment:
I believe we need to also validate `f.MasterAZ` with the same rules

Reply:
The recent commit simply copies the value of f.MasterAZ to f.ControlPlaneAZ, so validation for f.MasterAZ isn't necessary.